Wall/main.cpp

```diff
@@ -39,7 +39,7 @@ uint64_t glowInfoOffset;
 uint64_t LocalPlayerBase;
 uint64_t playerBase;
 
-uint64_t m_iGlowIndex       = 0xABF0;
+uint64_t m_iGlowIndex       = 0xA320;
```

Comment:
This is the windows offset. Please don't use any Offset dumpers from UC. They are all written for Windows. Since mac is fully 64bit the offsets will differ.